react/envelope_stack.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import {ReactComponent as ToDoFull} from './assets/stack_to-do_full.svg';
+import {ReactComponent as ToDoFull} from './assets/stack_to-do_half.svg';
```

Comment:
Do you want the ToDoFull component to take from stack_to-do_half.svg?

Reply:
Fixed